docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -45,7 +58,21 @@ Office Add-ins running on Windows can use the Debugger for Microsoft Edge extens
    > [!NOTE]
    > If you select **Cancel**, the dialog won't be shown again while this instance of the add-in is running. However, if you restart your add-in, you'll see the dialog again.
 
-1. You're now able to set breakpoints in your project's code and debug.
+   > [!IMPORTANT]
```

Comment:
Sorry, but I don't understand the comment. Is the IMPORTANT note what you mean by "this section"? It is already in step 5.

Reply:
Sorry, that was a typo on my end. I was suggesting to move it to Step 4.

Reply:
They won't see dialog until they press Open Taskpane, so it can't be moved to Step 4, but I adjusted it to have the reader stop debugging and then restart debugging.

Reply:
When I tested it, the dialog appeared when the host application started after pressing F5. I also noticed that the add-in didn't appear in the ribbon. Debugging appears to stop on its own, as the Stop Debugging option was grayed out after pressing Cancel in the dialog.

Reply:
I tested this a few times and I see what's happening. That error dialog appears anywhere from 5 to 35 seconds after I press F5 _regardless of what's happening in Office_. E.g., the error can come before the Office app has opened, after it has opened but before Show Taskpane is pressed, or after Show Taskpane is pressed. So, when the user sees it is going to depend on how fast Office opens and how fast the reader presses Show Taskpane. I think the best place to put this IMPORTANT note is in step 4, as you suggested, but it should say that the error could show up later.

Also, it does seem like the debugging stops automatically when I press cancel on the dialog most of the time, but not quite always, so I'll adjust the wording about manually stopping debugging.

Reply:
Very interesting. Thanks for retesting, Ricky. The changes you're proposing sound great.